DreamJournal/DreamJournal/NewEntryView.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  ContentView.swift
+//  DreamJournal
+//
+//  Created by CUBS Customer on 10/7/22.
+//
+
+import SwiftUI
+import CoreData
+
+struct CustomColor {
+    static let Navy = Color("Navy")
+    static let SkyPurple = Color("SkyPurple")
+    static let LavenderBox = Color("LavenderBox")
+}
+
+enum Category: String, Identifiable, CaseIterable {
+    var id: UUID {
```

Comment:
UUIDs are pretty cool to know about and use sometimes!

In this case, I might return a String of `rawValue` since all the items have a unique value already.

Actually though, there's another little trick that Swift supports and it seems a little odd, but you can often conform to `Identifiable` with this bit of code, and I think it always works with enums.

`var id: Self { return self }`

It looks a little funny, but people are often doing this nowadays because it doesn't really require any thinking about what to use as an ID. Not sure it matters in practice but just thought I'd share!

Reply:
That's a much shorter way of specifying the ID! Thank you for the tip, I'll definitely use this :)